api/src/main/java/org/openmrs/module/fhir2/api/dao/FhirDaoUtils.java

```diff
@@ -0,0 +1,24 @@
+package org.openmrs.module.fhir2.api.dao;
+
+import java.util.Date;
+
+import org.apache.commons.lang3.StringUtils;
+import org.hibernate.criterion.Criterion;
+import org.hibernate.criterion.Restrictions;
+
+public final class FhirDaoUtils {
+	
+	public static Criterion createActiveOrderCriterion(String path) {
+		if (StringUtils.isNotBlank(path)) {
+			path = path + ".";
+		}
+		
+		// ACTIVE = date activated null or less than or equal to current datetime, date stopped null or in the future, auto expire date null or in the future
```

Comment:
I based this on how activated was defined in Order.java, but happy to debate whether that's the right approach (or if my adaptaion is wrong)